utils/migration_tools/from_another_project/runs_migrator.py

```diff
@@ -0,0 +1,263 @@
+#
+# Copyright (c) 2023, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Instructions on how to use this script can be found at
+# https://github.com/neptune-ai/examples/blob/main/utils/migration_tools/from_another_project/README.md
+
+
+# %%
+import contextlib
+import logging
+import os
+import shutil
+import time
+from datetime import datetime
+from glob import glob
+from tempfile import TemporaryDirectory
+from typing import Optional
+
+import neptune
+import pandas as pd
+from neptune import management
+from neptune.exceptions import (
+    MissingFieldException,
+    TypeDoesNotSupportAttributeException,
+)
+from neptune.types import File, GitRef
+from tqdm.auto import tqdm
+
+# %%
+from_project = (
+    input("Enter project name to migrate from in WORKSPACE_NAME/PROJECT_NAME format:")
+    .strip()
+    .lower()
+)
+
+# %%
+to_project = (
```

Comment:
How about a flag to delete "from_project" runs/metadata once it's done copying?

Reply:
That would be a good addon. But given that it is already pretty easy to delete all the runs in a project, I would defer adding it to this script to a later version based on user feedback